plugins/policy-recommendation/antrea_crd.py

```diff
@@ -0,0 +1,627 @@
+#!/usr/bin/python3
+
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This library is used to define Antrea Network Policy related CRDs in Python.
+# Code structure is following the Kubernetes Python Client library (https://github.com/kubernetes-client/python).
+# This file could be changed to auto-generated by using openAPI generator in the future like the K8s python lib.
+
+class NetworkPolicy(object):
+    attribute_types = {
+        "kind": "string",
+        "api_version": "string",
+        "metadata": "kubernetes.client.V1ObjectMeta",
+        "spec": "NetworkPolicySpec",
+        "status": "NetworkPolicyStatus"
+    }
+
+    def __init__(self, kind=None, api_version=None, metadata=None, spec=None, status=None):
+        self.kind = kind
+        self.api_version = api_version
+        self.metadata = metadata
+        self.spec = spec
+        self.status = status
+
+    def to_dict(self):
```

Comment:
Is it possible to use inheritance to reuse the func to_dict?

Reply:
Yes, it's possible, for now, I'm following the code style of https://github.com/kubernetes-client/python/tree/master/kubernetes/client/models. They didn't involve the inheritance for the crds so I think we may don't need to do that too.